loop/management/commands/Outliers.py

```diff
@@ -0,0 +1,115 @@
+            daily_a_m_farmerShare_query_result = onrun_query(daily_a_m_farmerShare_query)
+            aggregator_wise_outliers = FSOC.data_Manipulator(daily_a_m_farmerShare_query_result, from_to_date)
+            worksheet_name = {'All': 'Farmer Share Outliers_' + str(from_to_date[0]) + "_" + str(from_to_date[1])}
+            file_to_send = header_dict_for_farmer_outlier['workbook_name'] % (
+            MEDIA_ROOT, '', str(from_to_date[0]), str(from_to_date[1]))
+
+            for elements in header_dict_for_farmer_outlier['column_properties']:
+                if 'data_type' in elements.keys() and elements['data_type']=='Date':
+                    date_format = workbook.add_format({'num_format': 'd mmm yy'})
+                    elements['format'] = date_format
+            table_properties = {'data': None, 'autofilter': False, 'banded_rows': False, 'style': 'Table Style Light 15',
+                                'columns': header_dict_for_farmer_outlier['column_properties']}
+
+
+        if case == 'TransportCost':
+            workbook = create_workbook(header_dict_for_transport_outlier['workbook_name'] % (
+            MEDIA_ROOT, '', str(from_to_date[0]), str(from_to_date[1])))
+
+            TSOC = TransportShareOutlierCompute.TransportCostOutlier()
+            daily_a_m_transportShare_query_result = onrun_query(daily_a_m_transport_share_query)
+            aggregator_wise_outliers = TSOC.data_Manipulator(daily_a_m_transportShare_query_result, from_to_date)
+            worksheet_name = {'All': 'Transport Cost Outliers_' + str(from_to_date[0]) + "_" + str(from_to_date[1])}
+            file_to_send = header_dict_for_transport_outlier['workbook_name'] % (
+            MEDIA_ROOT, '', str(from_to_date[0]), str(from_to_date[1]))
+            for elements in header_dict_for_transport_outlier['column_properties']:
+                if 'data_type' in elements.keys() and elements['data_type']=='Date':
+                    date_format = workbook.add_format({'num_format': 'd mmm yy'})
+                    elements['format'] = date_format
+            table_properties = {'data': None, 'autofilter': False, 'banded_rows': False, 'style': 'Table Style Light 15',
+                                'columns': header_dict_for_transport_outlier['column_properties']}
+
+        # Position of All is first as a co-incidence I think.
+        # Set correct column widths
+        # workbook = create_workbook('Farmer Share Outliers.xlsx')
+        # all_format = ['date_format']
+        # all_format_created = create_format(all_format, workbook)
+
+
+
+        # for aggregator in aggregators:
+        #     table_position_to_start = {'row': 2, 'col': 0}
+        #     worksheet_name[aggregator.name_en] = header_dict_for_farmer_outlier['worksheet_name'] % (
+        #     str(aggregator.name_en), str(from_to_date[0]), str(from_to_date[1]))
+
+        table_position_to_start = {'row': 2, 'col': 0}
```

Comment:
any of above mentioned IF cases would definitely be passed and no matter which case pass, this line has to be executed.